src/app/view/components/_common/site-selector/site-selector.component.html

```diff
@@ -16,4 +16,4 @@
     >
 </searchable-dropdown>
 
-<span *ngIf="totalRecords == 1" class="site-selector__title">{{currentSite && currentSite.value ? currentSite.value.hostname : '' | async}}</span>
\ No newline at end of file
+<span *ngIf="totalRecords == 1" class="site-selector__title">{{currentSite && currentSite['value'] ? currentSite['value'].hostname : '' | async}}</span>
```

Comment:
When doing this 'currentSite.value' it had a warning "Identifier value is not defined. "